hack/os-packages/rpm-fips-artifacts-getter/centos/7/containerd.fips.repo

```diff
@@ -0,0 +1,5 @@
+[containerd-fips]
+baseurl = https://containerd-fips.s3.us-east-2.amazonaws.com/7/x86_64
+gpgcheck = 0
+gpgkey = ""
```

Comment:
```suggestion
gpgkey = "https://containerd-fips.s3.us-east-2.amazonaws.com/7/x86_64/gpg.asc"
```

Reply:
https://github.com/mesosphere/konvoy/blob/76c918e77edd035ccb01e1b5ea5da8a607891841/ansible/group_vars/all/system.yaml#L112

Reply:
Thank you